app/src/main/java/nl/avans/drivioapp/adapter/AdvertisementAdapter.kt

```diff
@@ -45,6 +47,7 @@ class AdvertisementAdapter(
     }
 
     override fun onBindViewHolder(holder: AdvertisementViewHolder, position: Int) {
+        val sdf = SimpleDateFormat("yyyy/MM/dd", Locale.getDefault())
```

Comment:
Ik zou het variabel zelf een iets duidelijkere naam geven, dit in later gebruik duidelijker te maken waar het over gaat

Reply:
Deze is aangepast naar dateFormatter in de createAdvertisementFragment. In de adapter kon die verwijderd worden.